test/stream.js

```diff
@@ -17,6 +17,25 @@ test('buffer', async t => {
 	t.is(stdout.toString(), 'foo');
 });
 
+const checkEncoding = async (t, encoding) => {
+	const encodedString = '\u1000.';
```

Comment:
The string was chosen to give different outputs with each encoding type.

Reply:
I would write that as a short code comment as it's definitely not clear when just reading the code.

Reply:
Done in cff527d6698016392dbc92a2cac6832e82a5fb66 :+1: